client/dashboard/sites/overview/site-card.tsx

```diff
@@ -8,23 +9,24 @@ import {
 } from '@wordpress/components';
 import { dateI18n } from '@wordpress/date';
 import { __, sprintf } from '@wordpress/i18n';
+import { sitePHPVersionQuery } from '../../app/queries';
 import { getSiteStatusLabel } from '../../utils/site-status';
 import SitePreview from '../site-preview';
 import type { Site, Plan } from '../../data/types';
 
+function PHPVersion( { siteSlug }: { siteSlug: string } ) {
+	return (
+		( useQuery( sitePHPVersionQuery( siteSlug ) ).data && null ) ?? (
```

Comment:
Why `&& null` here, I guess it's a leftover from testing?

Reply:
Sorry my push failed, meant to remove